src/lib/molecules/AdvancedSearch/index.js

```diff
@@ -82,9 +87,11 @@ function AdvancedSearch({
 
     const inputAndPopoverWidthVariable = useMemo(
         () => ({
-            '--advanced-search-width': popoverOpen ? `${openedInputWidth}vw` : closedInputWidth
+            '--advanced-search-width': (isOpenControlled ? isOpen : popoverOpen)
```

Comment:
This logic will prevent to user pass the `openedInputWidth` and `closedInputWidth` props in case he controls the `isOpen` state. It’s not the right approach.

Reply:
Here are brackets witch will return the value to follow `isOpen` or `popoverOpen`